### PR TITLE
Using Grafana image instead of building it!

### PR DESCRIPTION
**Pull request by b3c3ll**
No need to build Grafana image in workflow or files

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -129,12 +129,14 @@ SRS_GNB_IP=172.22.0.37
 
 # GRAFANA
 GRAFANA_IP=172.22.0.39
+GRAFANA_USERNAME=open5gs
+GRAFANA_PASSWORD=open5gs
 
 # UE IPv4 Subnet Range for APN=internet
-UE_IPV4_INTERNET=192.168.100.0/24
+UE_IPV4_INTERNET=192.168.200.0/24
```
Comment (herlesupreeth) [inline .env:136]: please revert this change as it can vary from setup to setup

File: .env [hunk 2]
```diff
@@ -129,12 +129,14 @@ SRS_GNB_IP=172.22.0.37
...
 
 # UE IPv4 Subnet Range for APN=ims
-UE_IPV4_IMS=192.168.101.0/24
+UE_IPV4_IMS=192.168.201.0/24
```
Comment (herlesupreeth) [inline .env:139]: please revert this change as it can vary from setup to setup

File: 4g-external-ims-deploy.yaml [hunk 3]
```diff
@@ -1,4 +1,3 @@
-version: '3'
```
Comment (herlesupreeth) [inline 4g-external-ims-deploy.yaml:1]: Any reason why this was removed?

File: 4g-external-ims-deploy.yaml [hunk 4]
```diff
@@ -320,15 +319,23 @@ services:
...
       - ./grafana:/mnt/grafana
       - /etc/timezone:/etc/timezone:ro
       - /etc/localtime:/etc/localtime:ro
+    environment:
+      - GF_SECURITY_ADMIN_USER=${GRAFANA_USERNAME}
+      - GF_SECURITY_ADMIN_PASSWORD=${GRAFANA_PASSWORD}
+      #- GF_INSTALL_PLUGINS=${GRAFANA_INSTALL_PLUGINS}
+      - GF_PATHS_PROVISIONING=/etc/grafana/provisioning
+      - GF_PATHS_DATA=/var/lib/grafana
+      - METRICS_IP=${METRICS_IP}
```
Comment (herlesupreeth) [inline 4g-external-ims-deploy.yaml:338]: no need to pass METRICS_IP its already passed as part of env_file no?

File: 4g-external-ims-deploy.yaml (hunk 3 above)
Comment (b3c3ll) [reply 4g-external-ims-deploy.yaml:1]: Compose does not need the Version anymore!

Comment (b3c3ll) [reply 4g-external-ims-deploy.yaml:1]: You can see the warning the logs when you put it in there!

File: .env (hunk 2 above)
Comment (b3c3ll) [reply .env:139]: Tried to use different range as 192.168.201.0/24 as most of the ISP modems(ADSL, FIber, ...) and routers use 192.168.1.0/24 or 192.168.100.0/24!
So using 200 could solve the conflicts normally

if that goes wrong in your setup ... i'll revert it back to 100 and 101. Do i need so?

Comment (herlesupreeth) [reply .env:139]: please revert this..if it collides with the subnet of router users can change that in .env

File: 4g-external-ims-deploy.yaml (hunk 4 above)
Comment (herlesupreeth) [reply 4g-external-ims-deploy.yaml:338]: why? did it fail when it was not passed?

Comment (b3c3ll) [reply 4g-external-ims-deploy.yaml:338]: yup, tested and does not work properly!

**Comment by herlesupreeth** [comment]
Thank you for your contribution
